### PR TITLE
(DPS-2266) Improvements for Rating component

### PR DESCRIPTION
**Pull request by ale-kacperczyk**
## 🚀 Links

- [Feature branch](https://developers.labs.livechat.com/docs/feature/dps-2266)
- [Jira](https://livechatinc.atlassian.net/browse/DPS-2266)

## 📓 Description
Added a modal with an iframe displaying a Google Form.

### Features (for the website)

- New feature
- Breaking change

### CODE REVIEW

File: src/components/core/Rating/modal.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  margin: auto;
+  display: flex;
+  width: 100%;
+  align-items: center;
+  justify-content: center;
+  left: 0;
+  right: 0;
+  top: 0;
+  bottom: 0;
+  background-color: "white";
```
Comment (jakubsikora) [inline src/components/core/Rating/modal.js:28]:
```suggestion
  background-color: #FFFFFF;
```

File: src/components/core/Rating/modal.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  opacity: ${(props) => props.isLoading ? "1.0" : "0"};
```
Comment (jakubsikora) [inline src/components/core/Rating/modal.js:29]:
```suggestion
  opacity: ${(props) => props.isLoading ? 1 : 0};
```

File: src/components/core/Rating/modal.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+  return (
+    <div>
+      <ModalBase
+        onClose={onModalClose}
+        css={ModalBaseCss}
+      >
+        {isLoading
+          ? <LoaderWrapper isLoading={isLoading}><Loader size="large" /></LoaderWrapper>
+          : null}
```
Comment (jakubsikora) [inline src/components/core/Rating/modal.js:53]:
```suggestion
        {isLoading && <LoaderWrapper isLoading={isLoading}><Loader size="large" /></LoaderWrapper>}
```

**Review by jakubsikora — CHANGES_REQUESTED** [review]
Good work, few suggestions.

File: src/components/core/Rating/modal.js (hunk 2 above)
Comment (ale-kacperczyk) [reply src/components/core/Rating/modal.js:29]: deleted opacity

**Review by jakubsikora — APPROVED** [review]
Nicely done!

**Pull request merged**